Sources/ApolloCodegenLib/AST/SortedSelections.swift

```diff
@@ -47,59 +47,63 @@ struct SortedSelections: Equatable, CustomDebugStringConvertible {
 
   // MARK: Selection Merging
 
-  @inlinable mutating func mergeIn(_ selections: SortedSelections) {
```

Comment:
I agree this is not necessary - there's a good read on `@inlinable` over at https://swiftrocks.com/understanding-inlinable-in-swift.

Do you think there would be benefit in using `@inline` for `mergeIn(fragment:)` and `appendOrMerge<T: SelectionMergable>(selection:into:)`?